src/fusor/models.py

```diff
@@ -77,23 +71,24 @@ class FunctionalDomain(BaseModel):
                 "type": "FunctionalDomain",
                 "status": "lost",
                 "label": "Tyrosine-protein kinase, catalytic domain",
-                "_id": "interpro:IPR020635",
-                "associated_gene": {
+                "id": "interpro:IPR020635",
+                "gene": {
                     "id": "gene:NTRK1",
                     "gene_id": "hgnc:8031",
                     "label": "8031",
-                    "type": "GeneDescriptor",
+                    "type": "Gene",
                 },
-                "sequence_location": {
-                    "id": "fusor.location_descriptor:NP_002520.2",
-                    "type": "LocationDescriptor",
-                    "location": {
-                        "sequence_id": "ga4gh:SQ.vJvm06Wl5J7DXHynR9ksW7IK3_3jlFK6",
-                        "type": "SequenceLocation",
-                        "interval": {
-                            "start": {"type": "Number", "value": 510},
-                            "end": {"type": "Number", "value": 781},
-                        },
+                "sequenceLocation": {
+                    "id": "SL.vJvm06Wl5J7DXHynR9ksW7IK3_3jlFK6",
```

Comment:
I think GA4GH prefix is important to retain
```suggestion
                    "id": "ga4gh:SL.vJvm06Wl5J7DXHynR9ksW7IK3_3jlFK6",
```